errors/TypeSynonymInstance.md

```diff
@@ -3,19 +3,43 @@
 ## Example
 
 ```purescript
-module ShortFailingExample where
+module TypeSynonymInstanceError where
+import Prelude
 
-...
+type Mass = Number
+
+instance showMass :: Show Mass where
+  show mass = "Mass: " <> show mass <> "kg"
 ```
 
 ## Cause
 
-Explain why a user might see this error.
+*Type synonyms* are merely aliases (usually created to avoid verbose type annotations) and do not declare a new unique type. It is still possible to use the synonym and its equivalent type interchangeably:
+
+```purescript
+velocity = 123.456 :: Mass  -- oops
```

Comment:
Actually, I thought of velocity not as a number, but as another physical quanity. Any idea how to express that more clearly?